src/app/components/ATIAnalytics/canonical/sendBeaconOperaMiniScript.ts

```diff
@@ -1,12 +1,14 @@
 import isOperaProxy from '#app/lib/utilities/isOperaProxy';
 
 const sendBeaconOperaMiniScript = (atiPageViewUrlString: string) => `
-if (${isOperaProxy.toString()}()) {
-  var xhr = new XMLHttpRequest();
-  xhr.open("GET", "${atiPageViewUrlString}", true);
-  xhr.withCredentials = true;
-  xhr.send();
-} 
+    if (${isOperaProxy.toString()}() && !Boolean(window.hasOperaMinScriptRan)) {
+      window.hasOperaMinScriptRan = true;
```

Comment:
Still just a wee typo `Min` instead of `Mini`, but otherwise looks good. Think we should try this out and see if it works.